### PR TITLE
Adding visibility, invisibility and toBeClickable conditions.

### PR DESCRIPTION
**Pull request by vanilsonbr**
My team needed to adapt the java selenium code into node coe, and we needed support to three java existing conditions in our scripts: `ExpectedConditions::elementToBeClickable`, `ExpectedConditions::visibilityOfElementLocated` and `ExpectedConditions::invisibilityOfElementLocated` . I created an Until.js module extension to add those functions so that we didn't need too much changes on ours scripts. I decided to make this Pull Request to provide my code.

I was using async/await with those functions, but rewrited using Promise to follow the line of implementation that node's Until.js uses.

- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

---
This change is [Reviewable](https://reviewable.io/reviews/seleniumhq/selenium/6573)

### CODE REVIEW

File: javascript/node/selenium-webdriver/lib/until.js [hunk 1]
```diff
@@ -120,6 +126,31 @@ exports.alertIsPresent = function alertIsPresent() {
...
+ * Creates a condition for checking that an element is present on the DOM of a page and visible.
+ * Visibility means that the element is not only displayed but also has a height and width that is
+ * greater than 0.
+ *
+ * @param {!By} locator used to find the element
+ * @return {!Condition<WebElement>} the WebElement once it is located and visible
+*/
+exports.visibilityOfElementLocated = function(locator) {
+  return new Condition('visibility of element located by ' + locator.toString(), function(driver) {
+      return driver.findElement(locator).then(element => {
```
Comment (jleyba) [inline javascript/node/selenium-webdriver/lib/until.js]: could you use async/await to avoid promise chaining?

Comment (vanilsonbr) [reply javascript/node/selenium-webdriver/lib/until.js]: You mean only this statement or is it for the other changes too? The other functions use promise chaining too, even though they don't have 2 chains like this one.

**Comment by diemol** [comment]
@vanilsonbr, would you like to continue with this PR?

**Comment by diemol** [comment]
Closing as we did not get any reply.

**Pull request closed without merging**